test/lib/BedrockTester.cpp

```diff
@@ -583,4 +583,36 @@ int BedrockTester::waitForPort(int port) {
     return 1;
 }
 
+STable BedrockTester::getStatus(bool control) {
+    // FYI: sometimes the status command doesn't return with every key/value expected
+    SData request("Status");
+    return SParseJSONObject(executeWaitVerifyContent(request, "200", control));
+}
+
+string BedrockTester::getStatusTerm(string term, bool control) {
```

Comment:
still this should be waitForStatusTerm since it waits up until a timeout just like all of the other waitX functions.

Reply:
I think I found a way to make us both happy. Making "getStatusTerm" execute just once, and building retries into the "waitForCommit" command.

Reply:
This is likely to cause more build instability if other people use it without protecting against the fact that not every term comes back in `Status`.

Reply:
You were right. Created the "waitForStatusTerm" function.